Makefile

```diff
@@ -84,9 +91,10 @@ dcclean:
 	docker-compose down -v
 	docker image prune -f
 
+export COMPOSE_STUDIO_APP = ${COMPOSE_PROJECT_NAME}_studio-app_1
```

Comment:
ooh, I like this one!

Reply:
The person who wrote this knew what they were doing!